medusa/tv/episode.py

```diff
@@ -87,20 +84,209 @@
     get_scene_numbering,
     xem_refresh,
 )
-from medusa.tv.base import TV
-
-import shutil_custom
+from medusa.tv.base import Identifier, TV
```

Comment:
Leaving this here for now as I may implement an EpisodeIdentifier class that contains series ID info and episode numbering.